src/v/cluster/rm_group_proxy.h

```diff
@@ -25,7 +25,10 @@ class rm_group_proxy {
       kafka::group_id,
       model::producer_identity,
       model::tx_seq,
-      model::timeout_clock::duration)
+      model::timeout_clock::duration) // Timeout from begin_group_tx is not
+                                      // used. So we can pass usefull tx timeout
+                                      // to store it in group service for expire
```

Comment:
Also maybe move the comment to the top of the function declaration

Reply:
Since your patch is changing the semantics of what that function argument is, I suggested somewhere that we should document what that parameter is in the function definition. Just say that it is user set txn expiration timeout? Fine if you want to skip, it was a nit pick.